src/sensors/zjs_sensor_accel.c

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2017, Intel Corporation.
+
+#ifdef BUILD_MODULE_SENSOR_ACCEL
```

Comment:
Check whether this is really needed here now or not; I was able to remove it for GPIO because our build is now smart enough not to even build the file at all if it's not used. (Same with others.)